.github/workflows/test-visual-compare.yml

```diff
@@ -47,19 +47,24 @@ jobs:
           FIREBASE_CONFIG_TS: ${{ secrets.FIREBASE_CONFIG_TS }}
         run: echo $FIREBASE_CONFIG_TS > src/environments/firebaseConfig.ts
       - run: yarn install
+        # Currently all tests are run against global deployment as this contains debug sheets
+        # In the future we will want to move this to a custom deployment for testing purposes
       - name: Set Deployment
-        run: yarn scripts deployment set "$APP_DEPLOYMENT_NAME"
-        env:
-          APP_DEPLOYMENT_NAME: ${{ secrets.APP_DEPLOYMENT_NAME }}
+        run: yarn scripts deployment set "plh_global"
+        # Create a production build and serve via a local webserver
+        # HACK - as lifecycle_actions block template view simply remove them all
+      - name : Create production build
```

Comment:
Previously we would run against a local app server (e.g. `yarn start`) command, although for the process of testing a bit faster decided better to create a production build and just serve that locally instead